linker/Mono.Linker.Steps/MarkStep.cs

```diff
@@ -1198,5 +1202,23 @@ protected virtual void MarkInstruction (Instruction instruction)
 				break;
 			}
 		}
+
+		protected virtual bool ContinueWith (TypeDefinition type, TypeReference reference)
```

Comment:
Change this to be `HandleUnresolvedType (TypeReference reference)` and just throw